semgrep.jsonnet

```diff
@@ -9,21 +9,22 @@ local test_code_globs = ["Unit_*.ml", "Test_*.ml"];
 local less_important_code_globs = ["spacegrep/*", "experiments/*", "scripts/*"];
 
 local semgrep_rules = [
-  #TODO: this rule could be moved in pfff/semgrep.yml at some point
-  { id: "pfff-no-open-in",
-    match: { or: ["open_in_bin ...", "open_in ..."]},
-    # Same but using The old syntax:
-    #  "pattern-either": [
-    #    { pattern: "open_in_bin ..." },
-    #    { pattern: "open_in ..." },
-    #   ],
-    languages: ["ocaml"],
-    severity: "ERROR",
-    message: |||
-        It is easy to forget to close `open_in` with `close_in`.
-        Use `Common.with_open_infile()` instead.
-    |||,
-  },
+#TODO: does not parse anymore!!
+#  #TODO: this rule could be moved in pfff/semgrep.yml at some point
+#  { id: "pfff-no-open-in",
+#    match: { or: ["open_in_bin ...", "open_in ..."]},
```

Comment:
not sure what changed recently @brandonspark but this rule does not work anymore.
I've tried to replace the or: with any: but same problem.